cfitsio3490/docs/changes.txt

```diff
@@ -1,4 +1,58 @@
                    Log of Changes Made to CFITSIO
+		   
+Version 3.49 - Aug 2020
+
+  - Fix to imcompress.c.  It now turns off quantization if ZSCALE
+    and ZZERO columns are missing.  Treatment will be the same as
+    if ZQUANTIZ were set to 'NONE', even if ZQUANTIZ is present
+    and set to something else.
```

Comment:
@esheldon This is jogging my memory. Did you code around this in a recent bug fix?

Reply:
We recently added support for writing lossless compressed floating point images with setting the quantiz level to 0. And these files could be read by fitsio or astropy. But similar files written by astropy couldn’t be read by c/fitsio because of this bug. (I had thought it was a bug in astropy but it was this bug in cfitsio)